50_memory_augmented_transformers/python/model.py

```
@@ -0,0 +1,505 @@
+        self.d_model = config.d_model
+        self.n_neighbors = config.n_neighbors
+        self.temperature = config.temperature
+
+        # Projections
+        self.value_proj = nn.Linear(config.d_model, config.d_model)
+        self.out_proj = nn.Linear(config.d_model, config.d_model)
+        self.dropout = nn.Dropout(config.dropout)
+
+    def forward(
+        self,
+        x: torch.Tensor,
```

Ruff (0.14.13)

Unused method argument: `x`

(ARG002)